src/components/top_bar/top_bar.ts

```diff
@@ -250,7 +250,9 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
   }
 
   private openMenu(menu: Action, ev: MouseEvent) {
-    this.topBarToolStore.closeDropdowns();
+    if (this.topBarToolStore.currentDropdown) {
+      this.topBarToolStore.closeDropdowns();
```

Comment:
I think the inputs should be somewhat more resilient. Not sayning that we should not fix the calls here but any full render is a source of problem, which means any message received by the network is as well.

Reply:
Beside never using `t-att-value` I'm not sure what else we can do ...

Reply:
well it should write to a state and react at each onchange I gues, or we could imagine some sort of partial rendering / reorganise the component structure to have the sidepanel not subject to the hard render eveytime something occurs. @VincentSchippefilt  Maybe a good discussion for next week

Reply:
It's also a problem for the topbar font selection input, so not only the side panel. But yeah idk

Reply:
https://www.odoo.com/odoo/2328/tasks/4756754